src/Impostor.Api/Net/Messages/C2S/HandshakeC2S.cs

```diff
@@ -9,48 +9,15 @@ public static void Deserialize(IMessageReader reader, out GameVersion clientVers
             clientVersion = reader.ReadGameVersion();
             name = reader.ReadString();
 
-            if (clientVersion >= Version.V1)
-            {
-                reader.ReadUInt32(); // lastNonceReceived, always 0 since 2021.11.9
-            }
+            reader.ReadUInt32(); // lastNonceReceived, always 0 since 2021.11.9
```

Comment:
The reason the if-statements were in place is to help old clients realize that they need to update, and disconnect them with the proper message, instead of with a server exception